.pre-commit-config.yaml

```diff
@@ -29,15 +29,11 @@ repos:
     files: .
 
 # CloudFormation
-- repo: https://github.com/aws-cloudformation/cfn-python-lint
+- repo: https://github.com/aws-cloudformation/cfn-lint
   rev: v0.79.7
   hooks:
-  - id: cfn-python-lint
-    name: AWS CloudFormation Linter
+  - id: cfn-lint-rc
     files: code/solutions/.*\.(ya?ml|template)$
-    args:
-      - --ignore-templates=code/solutions/policy-as-code-with-guard/example_bucket_tests.yaml
-      - --ignore-templates=code/solutions/linting-and-testing/.taskcat.yml
```

Comment:
Are these ignore rules intended to be removed?

Reply:
Yes, we have `.cfnlintrc` file which will now be used everywhere rather than duplicating it in multiple places...